golang1.17/lib/launcher.go

```diff
@@ -117,8 +119,16 @@ func execute(f interface{}, in []byte) ([]byte, error) {
 		}
 	}
 
+	// Setup a context that cancels at the given deadline.
+	deadlineMillis, err := strconv.ParseInt(os.Getenv("__OW_DEADLINE"), 10, 64)
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse deadline: %w", err)
+	}
+	ctx, cancel := context.WithDeadline(context.Background(), time.UnixMilli(deadlineMillis))
```

Comment:
If this fails, should we use a default value or skip the deadline addition to the context all together?

Reply:
Yeah, I just had an occurence in a test that actually triggers this. I think we should default to `Background()` if the env var is empty.